client/allocrunner/tasklifecycle/doc.go

```diff
@@ -1,3 +1,4 @@
+//nolint
```

Comment:
```suggestion
```

Ahh this was fixed in https://github.com/hashicorp/nomad/pull/14232, so you can remove this 👍